extensions/csv/src/main/java/io/deephaven/csv/parsers/BooleanParser.java

```diff
@@ -0,0 +1,51 @@
+package io.deephaven.csv.parsers;
+
+import io.deephaven.csv.sinks.Sink;
+import io.deephaven.csv.tokenization.Tokenizer;
+import io.deephaven.csv.parsers.context.ParseContext;
+
+public final class BooleanParser extends ParserBase<byte[]> {
+    public static BooleanParser INSTANCE = new BooleanParser();
+
+    private BooleanParser() {}
+
+    @Override
+    public Sink<byte[]> tryParse(ParseContext ctx, IteratorHolder ih, IteratorHolder ihAlt) {
+        return twoPhaseDriver(ctx, ih, ihAlt, BooleanParser::tryParseHelper,
+                ctx.sinkFactory::makeBooleanAsByteSink);
+    }
+
+    private static boolean tryParseHelper(ParseContext ctx, IteratorHolder ih,
```

Comment:
For me this crosses the threshold for value for `final` in method arguments.   My proposed threshold is "at a glance I can't see all the uses of the arguments in the body".  Typically because the body is longer that just a few lines of code that I can have in my field of view together with the method signature.

Reply:
It will be my new goal to put "final" everywhere I can. If I miss some, kindly let me know :-)